src/components/model-form/model-form.js

```diff
@@ -53,6 +53,13 @@ const ModelForm = ({ model, id, isEdit }) => {
     modelImage,
     setModelImage
   } = useModelHandlers();
+
+  useEffect(() => {
+    if(!isEdit){
+      dispatch(getCategories())
+    }
+  },[])
+
   const languageTabs =
     languages.length > 0
       ? languages.map((lang, index) => <Tab label={lang} key={lang} />)
```

Comment:
same

Reply:
This Tabs isn't use ane more replece it on LanguagePanel